src/renderer/views/Watch/Watch.js

```diff
@@ -321,7 +327,11 @@ export default defineComponent({
         this.getVideoInformationLocal()
       }
 
+      document.removeEventListener('click', this.resetAutoplayInterruptionTimeout)
+      document.addEventListener('click', this.resetAutoplayInterruptionTimeout)
```

Comment:
I am wondering what happen to people interacting with keyboards only...

Reply:
Great callout, addressed now